configure.ac

```diff
@@ -85,7 +85,8 @@ AC_PATH_TOOL(STRIP, strip)
 AC_PATH_TOOL(GCOV, gcov)
 AC_PATH_PROG(LCOV, lcov)
 dnl Python 3.x is supported from 3.4 on (see https://github.com/bitcoin/bitcoin/issues/7893)
-AC_PATH_PROGS([PYTHON], [python3.7 python3.6 python3.5 python3.4 python3 python])
+dnl Python 3.4 is specified in .python-version and should be used if available (see https://github.com/bitcoin/bitcoin/pull/15285)
+AC_PATH_PROGS([PYTHON], [python3.4 python3.7 python3.6 python3.5 python3 python])
```

Comment:
nit: Could do without the github link above and sort like `3.4 3.5 3.6 ... 3`. Feel free to ignore